src/vip/batch_address_test_tool/processor.clj

```diff
@@ -63,7 +63,10 @@
 
 (defn retrieve-polling-locations*
   [ctx]
-  ctx)
+  (let [addresses (:addresses ctx)
+        polling-locations (doall (map #(civic-info/address->polling-location (:address %)) addresses))
+        merged (mapv #(assoc %1 :api-result %2) addresses polling-locations)]
```

Comment:
And here too, unless this _needs_ to be a vector of polling locations (is the order significant?), `map` would be a good substitute for `mapv` because it's being put right into the returned data structure on the next line (as opposed to performing some kind of side effect that you want to make sure happens for each element of the mapped-over collection).